dashboard/app/views/pd/workshop_mailer/_supply_list.html.haml

```diff
@@ -18,5 +18,5 @@
       do not have one available, one will be provided for you at the workshop.
   - if @workshop.course == Pd::Workshop::COURSE_CSF && @is_enrollment_receipt
     %li Your curiosity and readiness to learn!
-  - unless @workshop.teachercon? || @workshop.summer? || @workshop.fit_weekend?
+  - unless @workshop.teachercon? || @workshop.summer? || @workshop.fit_weekend? || @workshop.course == Pd::Workshop::COURSE_CSF
```

Comment:
you can remove the reference to teachercon while you're in here, since there won't be any future teachercons.

Reply:
fixed